lite-locals.tf

```diff
@@ -1,5 +1,63 @@
+output "thor_max_jobs" {
+   value = var.thor_max_jobs
+}
+output "thor_num_workers" {
+   value = var.thor_num_workers
+}
+output "thor_node_size" {
+   value = var.aks_node_sizes.thor
+}
+output "thor_ns_spec" {
+   value = local.ns_spec[var.aks_node_sizes.thor]
+}
+output "thor_worker_cpus" {
+   value = var.thor_worker_cpus
+}
+output "thorWorkersPerNode" {
+   value = "local.ns_spec[${var.aks_node_sizes.thor}].cpu / var.thor_worker_cpus = ${local.thorWorkersPerNode}"
+}
+output "thor_worker_ram" {
+   value = "local.ns_spec[${var.aks_node_sizes.thor}].ram / local.thorWorkersPerNode = ${local.thor_worker_ram}"
+}
+output "nodesPer1Job" {
+   value = "var.thor_num_workers /  local.thorWorkersPerNode = ${local.nodesPer1Job}"
+}
+output "thorpool_max_capacity" {
+   value = "local.nodesPer1Job * var.thor_max_jobs = ${local.thorpool_max_capacity}"
+}
 locals {
-  helm_chart_timeout=600
+  ns_spec = {
```

Comment:
Where is this information mapped back to Azure VM size names?

Reply:
I'm not sure what you are asking?? Are you saying that the value of 'helm_chart_timeout' should be dependent on the VM size??

Reply:
I am asking about how VM sizing is now working.  It used to be that we had to choose a specific VM size and version, such as Standard_D8_v4.  Do we not need that any longer?

Reply:
In the repo: https://github.com/hpccsystems-solutions-lab/tlh-oss-terraform-azurerm-aks
here is where xlarge is translated into Standard_D4s_v4, for example: modules/node-groups/modules/node-group/local.tf